File: src/main/java/net/andrecarbajal/naviMusic/NaviMusicApplication.java
package net.andrecarbajal.naviMusic;

import net.andrecarbajal.naviMusic.manager.GUIManager;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class NaviMusicApplication {
    private static final GUIManager guiManager = new GUIManager();

    public static void main(String[] args) {
        guiManager.checkGui(args);

        SpringApplication.run(NaviMusicApplication.class, args);
    }
}

File: src/main/kotlin/net/andrecarbajal/naviMusic/NaviMusicApplication.kt
package net.andrecarbajal.naviMusic

import net.andrecarbajal.naviMusic.ui.UIManager
import org.springframework.beans.factory.getBean
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.ApplicationContext

@SpringBootApplication
class NaviMusicApplication

fun main(args: Array<String>) {
    val context: ApplicationContext = SpringApplication.run(NaviMusicApplication::class.java, *args)

    val uiManager = context.getBean<UIManager>()
    uiManager.checkGui(args)
}
